packages/nouns-subgraph/src/stream-escrow.ts

```diff
@@ -0,0 +1,117 @@
+import { ETHStreamed, Stream, StreamFastforward } from './types/schema';
+import {
+  AllowedToCreateStreamChanged,
+  DAOExecutorAddressSet,
+  ETHRecipientSet,
+  ETHStreamedToDAO,
+  NounsRecipientSet,
+  StreamCanceled,
+  StreamCreated,
+  StreamFastForwarded,
+  StreamsForwarded,
+} from './types/StreamEscrow/StreamEscrow';
+import { BIGINT_ZERO } from './utils/constants';
+import {
+  genericUniqueId,
+  getCurrentStream,
+  getOrCreateStreamCreationPermission,
+  getOrCreateStreamsOfNoun,
+  getStreamEscrowState,
+} from './utils/helpers';
+
+export function handleETHStreamedToDAO(event: ETHStreamedToDAO): void {
+  const es = new ETHStreamed(genericUniqueId(event));
+  es.amount = event.params.amount;
+  es.createdTimestamp = event.block.timestamp;
+  es.createdBlock = event.block.number;
+  es.save();
+
+  const s = getStreamEscrowState();
+  s.totalAmountStreamedToDAO = s.totalAmountStreamedToDAO.plus(event.params.amount);
+  s.save();
+}
+
+export function handleStreamCreated(event: StreamCreated): void {
+  const nounId = event.params.nounId.toString();
+  const streamId = genericUniqueId(event);
+
+  const s = new Stream(streamId);
```

Comment:
@eladmallel perhaps we can also save the currentTick here as the startTick?

Reply:
fixed: https://github.com/nounsDAO/nouns-monorepo/pull/869/commits/47394bdf71a82e2d4136e0be189eccc2bb795d08